### PR TITLE
Php1branch > tl1revbranch

### PR DESCRIPTION
**Pull request by php-1**

### CODE REVIEW

File: portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php [hunk 1]
```diff
@@ -442,7 +442,7 @@ public function __construct($db)
 								'titre'=>'List Agents',
 								'mainmenu'=>'ihcm',
 								//'leftmenu'=>'ihcm',
-								'url'=>'/ihcm/social_insurance/social_listAgent.php?action=create',
+								'url'=>'/ihcm/social_insurance/social_listAgent.php',
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php]: don't make a file names in capital letters please.

File: portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php [hunk 2]
```diff
@@ -398,7 +398,7 @@ public function __construct($db)
 								'titre'=>'List Of Assign',
 								'mainmenu'=>'ihcm',
 								//'leftmenu'=>'ihcm',
-								'url'=>'/ihcm/training/training_listAssign.php?action=create',
+								'url'=>'/ihcm/training/training_listAssign.php',
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php]: don't make a file names in capital letters please.

File: portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php [hunk 3]
```diff
@@ -472,7 +472,7 @@ public function __construct($db)
 								'titre'=>'List Assigned Insurance',
 								'mainmenu'=>'ihcm',
 								//'leftmenu'=>'ihcm',
-								'url'=>'/ihcm/social_insurance/social_listAssign.php?action=create',
+								'url'=>'/ihcm/social_insurance/social_listAssign.php',
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php]: don't make a file names in capital letters please.

File: portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/sql/training/llx_hcm_training_assign_statuslog.sql [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ * and open the template in the editor.
+ */
+/**
+ * Author:  php1
+ * Created: Jan 23, 2018
+ */
+
+CREATE TABLE llx_hcm_training_assign_statuslog (
+  rowid INTEGER(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  user_id INTEGER(11) NOT NULL,    -- login user
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/sql/training/llx_hcm_training_assign_statuslog.sql]: userid

File: portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/sql/training/llx_hcm_training_statuslog.sql [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+ * and open the template in the editor.
+ */
+/**
+ * Author:  php1
+ * Created: Jan 23, 2018
+ */
+
+CREATE TABLE llx_hcm_training_statuslog (
+  rowid INTEGER(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  user_id INTEGER(11) NOT NULL,
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/sql/training/llx_hcm_training_statuslog.sql]: userid

File: portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php (hunk 2 above)
Comment (php-1) [reply portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php]: dolibarr called name after mod ==> modIHCM

File: portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php (hunk 1 above)
Comment (php-1) [reply portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php]: ok

File: portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/sql/training/llx_hcm_training_statuslog.sql (hunk 5 above)
Comment (php-1) [reply portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/sql/training/llx_hcm_training_statuslog.sql]: user_id ==> fk_user???

File: portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/sql/training/llx_hcm_training_assign_statuslog.sql (hunk 4 above)
Comment (php-1) [reply portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/sql/training/llx_hcm_training_assign_statuslog.sql]: fk user

File: portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php (hunk 3 above)
Comment (php-1) [reply portfolio/ifintech/v1/ierp/htdocs/custom/ihcm/core/modules/modIHCM.class.php]: ok

**Pull request merged**